### ISSUE TITLE
Router Level selection dropdown in 3.0 and up

### ISSUE THREAD

**Issue by ddn4t** [bitfocus/companion-module-generic-swp08#18]
The Generic: SW-P-08 router module has had the drop down to select router levels broken in 3.0 and up.

The module itself seems to be working properly outside of that still, as though the drop down is still broken on them, buttons made prior to the upgrade to 3.0 and up still work properly. The levels that were selected on the button during its construction pre-3.0 are still technically "selected" when the button is pressed though you can not see them.

**Comment by daniep01** [comment]
Thanks for reporting this, sorry the module hasn't been re-written for v3 but it's quite a big task.

**Comment by ddn4t** [comment]
I have no doubt, 3.0 was a huge change. 

I posted the same message in "Modules with compatibility issues" issue and they found that they replied a little before this saying that the config field type was missed in the compatibility layer and that it should work in the next beta build.

So if that is the case, that should buy a bit more time. I used the module quite a bit and have not noticed any huge issues that popped up with the move to 3.0
